wrappers/windres-wrapper.c

```diff
@@ -266,7 +278,12 @@ int _tmain(int argc, TCHAR* argv[]) {
     else if (!_tcscmp(arch, _T("aarch64")))
         machine = _T("ARM64");
 
-    const TCHAR *CC = concat(target, _T("-clang"));
+    const TCHAR *resolved_CC =
+        concat(dir, concat(target, _T("-") _T(CC)));
+    const TCHAR *resolved_LLVM_RC =
+        concat(dir, concat(target, _T("-") _T(LLVM_RC)));
```

Comment:
We don't prefix llvm-rc or llvm-cvtres with the target triple, as there's no triple-prefixed wrapper/symlink for those, as one doesn't normally call them directly, and all target specific behaviour in them is handled by the `-machine:` parameter to llvm-cvtres. It's only used for clang (where it's essential to find headers in the right sysroot, when preprocessing).

And I'm still curious how this patch allows you to reach your intended goal of baking in an absolute path (as building with `-DCC=/absolute/path/to/clang` wouldn't work in this form).

Reply:
So we have wrapper scripts around the exes which have some flags, and everything is prefixed. I realized I didn't actually need the absolute paths, just the prefixes.

I should retitle this PR, and add some CPP to accommodate both our usecases.